src/seq_mv/csr_spgemm_device.h

```diff
@@ -22,13 +22,19 @@ constexpr HYPRE_Int SYMBL_HASH_SIZE[11] = { 0, 32, 64, 128, 256, 512, 1024, 2048
 constexpr HYPRE_Int NUMER_HASH_SIZE[11] = { 0, 16, 32,  64, 128, 256,  512, 1024, 2048, 4096,  8192 };
 #elif defined(HYPRE_USING_HIP)
 constexpr HYPRE_Int NUMER_HASH_SIZE[11] = { 0,  8, 16,  32,  64, 128,  256,  512, 1024, 2048,  4096 };
+/* WM: todo - what should this be for intel? */
```

Comment:
Probably need to tune this with the information of Intel GPU shared memory size, etc.

Reply:
Yes, I think there are a couple places where parameters need to be optimized for Intel hardware. But I plan to leave performance optimization for a future PR.